testsuites/syncgateway/functional/tests/test_db_online_offline_resync.py

```diff
@@ -305,11 +305,18 @@ def test_bucket_online_offline_resync_with_online(params_from_base_test_setup, s
         if resync_occured:
             break
 
-    time.sleep(10)
-
-    status = sg_client.bring_db_online(cluster_conf=cluster_conf, db="db")
-    assert status == 0
-    log_info("online request issued !!!!! response status: {}".format(status))
+    retries = 0
+    while retries < 3:
+        try:
+            status = sg_client.bring_db_online(cluster_conf=cluster_conf, db="db")
+            assert status == 0
+            log_info("online request issued !!!!! response status: {}".format(status))
+            break
+        except AssertionError as error:
+            retries = retries + 1
+            time.sleep(30)
```

Comment:
this seems wait much longer than the original 10 sec. are you able to increase the retries cap and make the sleep period shorter? in that case, you will end up wait for just the time you need to wait.

Reply:
In other platforms we needed more sleep in both places.

Reply:
actually this test is failing only for Mac as it is slow. All other platforms does not have issue. if Mac is taking that long like 90 seconds, can you have retries for 10 times and have sleep for 10 seconds. It is better not to add additional 20 seconds for other platforms if it is not necessaary